src/logic/calculate.js

```diff
@@ -0,0 +1,25 @@
+import operate from './operate';
+
+// data has mock information so the app won't crash
+const Calculate = (data = { total: null, next: null, operation: null }, btnName = '+') => {
+  let { total, next, operation } = data;
+
+  if (btnName === 'AC') {
+    total = 0;
+    next = 0;
+    operation = '';
+  } else if (btnName === '+/-') {
+    total *= -1;
+    next *= -1;
+  } else if (['÷', 'X', '+', '-', '%'].includes(btnName)) {
```

Comment:
- [ ] The `modifier operator (%)` should be handled in the `operate.js` module.

Screenshot from requirement number 9 on pathwright 👇🏾 

<img width="1362" alt="Screen Shot 2021-03-16 at 8 58 09 PM" src="https://user-images.githubusercontent.com/48804258/111371935-525f6080-869a-11eb-8a5e-ec8aed4035e7.png">